src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,195 @@
+
+  * Visualize and analyze your entire stack in one unified experience, with AI-powered analytics to find and fix issues fast.
+
+  * Pay only for what you use.
+
+<img
+  title="New Relic main page"
+  alt="New Relic main page"
+  src={NROneMain}
+/>
+
+
+
+# Let's start today! [#start-today]
+
+<Steps>
+  <Step>
+    ## Sign up for New Relic
+
+    Are you ready?
+
+    <ButtonGroup>
+      <ButtonLink
+        role="button"
+        to="https://newrelic.com/signup"
+        variant="primary"
+      >
+        Sign up for New Relic
+      </ButtonLink>
+    </ButtonGroup>
+
+    Does your organization already have New Relic but you're just adding a new team or project? See [Add a new account](/docs/accounts/accounts-billing/account-structure/add-accounts).
+  </Step>
+  <Step>
+    ## Ingest some data
+
+    Follow our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==) to auto-discover your environment. The guided install walks you through a recommended setup. This step-by-step process makes the installation easier. It ensures that all needed parts are set up right.
+
+  </Step>
+
+  <Step>
+    ## Check out some dashboards
+
+    Explore your environment in real time with some out-of-the-box UI views. You'll see data in the **Infrastructure**, **APM**, and **Logs** UI pages. 
+
+    Learn how to [get started with dasboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/).
+  </Step>
+
+  <Step>
+    ## Add more data
+
+    If you need data from other sources, check our [quickstarts](https://newrelic.com/instant-observability). These quickstarts include step-by-step instructions to guide you through the setup process. You'll be able to monitor your systems in no time. With our quickstarts, monitoring becomes effortless.
+  </Step>
+
+  <Step>
+    ## Set up alerting
+
+    You can manage your <InlinePopover type="alerts" />, [create](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/) new ones, configure your [notifications](/docs/alerts-applied-intelligence/notifications/intro-notifications/), identify [anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence/), configure [correlation policies](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions), and more. 
+
+    Some of the [quickstarts](https://newrelic.com/instant-observability) include pre-configured alerts that save you time setting up monitoring tools.
+
+    You can also check [our tutorial](/docs/tutorial-create-alerts/create-new-relic-alerts/) on getting started with New Relic alerts.
+    
+  </Step>
+
+  <Step>
+
+    ## Add users
+
```

Comment:
I'm not sure we need this section.... adding users is kind of an edge case in a way.

Reply:
Removed ## Add users